avocado/core/test.py

```diff
@@ -961,15 +957,14 @@ def test(self):
         raise exceptions.NotATestError(e_msg)
 
 
-class SkipTest(Test):
+class FakeTest(Test):
```

Comment:
Btw the `MissingTest` and such can safely be simply inherited from `object` as they are never executed. Anyway I'd prefer if you did not touch those (MissingTest and NotATest) as I'll be moving them (and inheriting them from object) in loaders cleanup.